### ISSUE TITLE
Forms stuck "In Maintenance"

### ISSUE THREAD

**Issue by tim1mw** [fredsdc/redmine_web_forms#2]
I'm struggling to work out what I need to do to make a new form "active". I've followed the instructions and reviewed the settings several times, but the form is stubbornly stuck "In maintenance" when I try to access it. The instructions imply that as long as the user has permission to create an issue, the form should work, I'm testing as admin and my admin user is a member of the project, so definitely has permission.

**Comment by fredsdc** [comment]
This plugin ignores some of the admin priviledges. The code to validate webform is in the model:

```
 def validate_webform(user=User.current)
    roles = get_user_roles(user)

    self.project.present? &&
    self.tracker.present? &&
    self.project.trackers.include?(self.tracker) &&
    self.issue_status.present? &&
    roles.map{|role| can_add_issue(role)}.any? &&
    WorkflowTransition.where(
      old_status_id: 0,
      tracker_id: self.tracker_id,
      role_id: roles.map{|r| r.id}
    ).map{|w| w.new_status_id}.include?(self.issue_status_id)
  end
```

It states that:
- the project, the tracker and the issue status must exist and be defined
- the project must have the tracker enabled
- one of the user roles on the project must be able to create an issue
- there is a workflow from new issue to the issue status for the tracker/role

**Comment by tim1mw** [comment]
I believe I followed all those steps, but it was still stuck in maintenance, with no clue being given as to what I had missed. It would be handy if instead of just saying "in maintenance" there was a furher message indicating which of the vital steps needed to make this plugin work had not been undertaken correctly.

However, I went with a different plugin in the end since I couldn't get this to work, so this is no longer a priority for me.
